qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/DeploymentClusteredTest.java

```diff
@@ -126,8 +126,7 @@ public void shouldRedistributeDeploymentWhenDeploymentPartitionIsRestarted() {
     final var processDefinitionKey = deploymentEvent.getKey();
 
     assertThat(
-            RecordingExporter.deploymentDistributionRecords()
-                .withIntent(DeploymentDistributionIntent.COMPLETED)
+            RecordingExporter.deploymentRecords(DeploymentIntent.CREATED)
```

Comment:
❌ These checks are not equivalent. Let's use `CommandDistribution.ACKNOWLEDGED` (this means the other partition has received it and the distributing partition is aware of this. This scenario should be adjusted for several cases in a83cd854b114bd26f8308fccacf30523d3075b03